cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -111,6 +115,14 @@ public static void main(final String[] args) {
                 writeJSONfile(bom, fileName);
                 break;
 
+            case "signSBOM":
+                bom = signBom();
+                JSONObjectWriter writer = new JSONObjectWriter();
+                JSONAsymKeySigner signer = new JSONAsymKeySigner(privateKey, publicKey);
```

Comment:
privateKey and publicKey don't look to be defined, they need to be java.security.PrivateKey

Reply:
Sample code:
```
writer.setSignature(new JSONAsymKeySigner(privateKey).setPublicKey(publicKey));
```

Reply:
For the moment assume there will be string representation param to the --signSBOM \<privateRSAKey text\>